2/contrib/openshift/base-plugins.txt

```diff
@@ -21,7 +20,7 @@ junit:1.30
 kubernetes:1.29.7
 kubernetes-client-api:4.13.3-1
 lockable-resources:2.10
-mailer:1.32.1
+mailer:1.34
```

Comment:
On 4.6 and 4.7 we currently have mailer 1.32.1. Are we broken on these versions as well?